library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -783,7 +801,7 @@ private void dropPointOnMap(@NonNull LatLng latLng) {
         dropPointOnMap(latLng, null);
     }
 
-    private void dropPointOnMap(@NonNull LatLng latLng, @Nullable MarkerOptions markerOptionsParam) {
+    public void dropPointOnMap(@NonNull LatLng latLng, @Nullable MarkerOptions markerOptionsParam) {
```

Comment:
Consider using the method `KujakuMapView#updateDroppedPoints` in place of making this method public.

It performs checks to ensure points are not duplicated and is already public. It also takes care of the case where the `MapboxMap` is null.

Reply:
Thanks, i will update the code to use method `KujakuMapView#updateDroppedPoints`